include/uv/win.h

```diff
@@ -265,19 +265,8 @@ typedef union {
 
 typedef union {
   struct {
-    unsigned int num_readers_;
-    CRITICAL_SECTION num_readers_lock_;
-    HANDLE write_semaphore_;
+    SRWLOCK read_write_lock_;
   } state_;
-  /* TODO: remove me in v2.x. */
-  struct {
-    SRWLOCK unused_;
-  } unused1_;
-  /* TODO: remove me in v2.x. */
-  struct {
-    uv_mutex_t unused1_;
-    uv_mutex_t unused2_;
-  } unused2_;
 } uv_rwlock_t;
```

Comment:
This is part of the public API (and ABI) so can you make sure the size of the struct doesn't change?

Reply:
I can make sure `uv_rwlock_t` does not change in size.  Do I also need to worry about the inner `state_` struct size?

Reply:
No, just the outer struct is fine.

Reply:
I added enough padding to bring it back to parity with the previous size.  It took 72 bytes(!) of padding.  I also added a `STATIC_ASSERT` to the .c file so that this is guarded in the future.